ElectronClient/app/gui/MainScreen.jsx

```diff
@@ -137,15 +144,15 @@ class MainScreenComponent extends React.Component {
 							}
 						}
 
-						this.setState({ promptOptions: null });
+						this.setState({promptOptions: null});
 					},
 				},
 			});
 		} else if (command.name === 'setTags') {
 			const tags = await Tag.tagsByNoteId(command.noteId);
 			const noteTags = tags
 				.map(a => {
-					return { value: a.id, label: a.title };
+					return {value: a.id, label: a.title};
```

Comment:
I don't understand why this is being reformatted, or why it's not picked by the linter as the preferred syntax would be the one that's been deleted.

Reply:
I've disabled / uninstalled all of the linters in my editor that i know of. I'm not even touching any parts outside of what I'm currently working on.

Reply:
That was my mistake actually, I thought there was a rule for this but there wasn't. So I've just added one, as well as for array spacing.

Reply:
Ale is a universal linter that works for almost all languages. Of course, i did notice that the eslint script is only using the default linter settings and can be custom configured using the `--config` or `--conf` flags. Maybe the defaults changed too?

Reply:
Despite this, I'm always up for trimming my plugins and configs at times. I need to nuke my neovim config a bit. I have quite a bit of junk in it and coc.nvim is handling most of my coding needs. VsCode is a hog on my system but i love Intellisense and some of its plugins. Coc adds full support of both.